src/docs/Glossary.adoc

```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
+Job Analytics:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of executed workflows along with their input variables and results.
+
+[[_glossary_node_gantt]]
+Node Gantt:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of <<_glossary_proactive_node,*ProActive Nodes*>> usage over time.
+
+[[_glossary_analytics_portal]]
+Analytics Portal:: Web interface of the <<_glossary_job_analytics,*Analytics*>> component.
 
 [[_glossary_notification_service]]
 Notification Service:: ProActive component that allows a user or a group of users to subscribe to different types of notifications (web, email, sms or a third-party notification) when certain event(s) occur (e.g., job In-Error state, job in Finished state, scheduler in Paused state, etc.).
 
 [[_glossary_notification_portal]]
-Notification Portal::
-ProActive portal to visualize notifications generated by the <<_glossary_notification_service,*Notification Service*>> and manage subscriptions.
+Notification Portal:: Web interface to visualize notifications generated by the <<_glossary_notification_service,*Notification Service*>> and manage subscriptions.
 
 [[_glossary_notification_subscription]]
-Notification Subscription::
-Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
+Notification Subscription:: Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
 
 [[_glossary_notification_third_party]]
-Third Party Notification::
-Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
+Third Party Notification:: Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
 
 [[_glossary_notification_methods]]
-Notification Method::
-Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
+Notification Method:: Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
 
 [[_glossary_cloud_automation_service]]
 Cloud Automation Service:: ProActive component that allows a user to easily manage any On-Demand Services (PaaS, IaaS and MaaS) with full Life-Cycle Management (create, deploy, pause, resume and terminate).
 
+[[_glossary_cloud_automation_portal]]
+Cloud Automation Portal:: Web interface of the <<_glossary_cloud_automation_service,*Cloud Automation service*>>.
+
+[[_glossary_workflow_automation_portal]]
+Workflow Automation Portal:: Web interface which provides an end-user portal to submit workflows, monitor their activity and recieve workflow results.
```

Comment:
monitor their activit**ies**

Reply:
plural is not necessary here. Several workflows, each as an activity, so I can say their activity.